000-vip-init.php

```diff
@@ -139,6 +139,8 @@
 	),
 );
 
+define( 'CRON_CONTROL_ADDITIONAL_INTERNAL_EVENTS', $internal_cron_events );
```

Comment:
Do we need to move this line up or is it fine to simply delete from line 152 to 158?

Reply:
Moved it up a few lines since the JP-related constants no longer affect it, just so it's closer to it's source. Left $internal_cron_events as an array though in case there is need in the future to add to it further based on conditions.